clerk/organizations.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+from clerk import types
+from clerk.client import Service
+
+
+class OrganizationsService(Service):
+    endpoint = "organizations"
+
+    async def list(self) -> List[types.Organization]:
+        """Retrieve a list of all organizations"""
+        async with self._client.get(self.endpoint) as r:
```

Comment:
```suggestion
        async with self._client.get(self.endpoint) as resp:
```

let's use `resp` instead of `r` for all of these methods

Reply:
Please update the rest of the methods and make sure the code still works. There's still a reference to `s` rather than `resp_json`

Reply:
Sorry about that, should have tested it.
Not a big user of one-letter vars myself, I kept them originally to be "coherent" with your methods in clients.py